Source/Cells/Popover/FORMPopoverFieldCell.m

```diff
@@ -137,12 +126,23 @@ - (void)titleLabelPressed:(FORMFieldValueLabel *)titleLabel {
 
     [self updateContentViewController:self.contentViewController withField:self.field];
 
-    if (!self.popoverController.isPopoverVisible) {
-        [self.popoverController presentPopoverFromRect:self.bounds
-                                                inView:self
-                              permittedArrowDirections:UIPopoverArrowDirectionUp | UIPopoverArrowDirectionDown
-                                              animated:YES];
+    self.contentViewController.modalPresentationStyle = UIModalPresentationPopover;
+    UIPopoverPresentationController *presentationController = [self.contentViewController popoverPresentationController];
+    presentationController.sourceView = self;
+    presentationController.sourceRect = self.bounds;
+
+    UIViewController *topViewController = [self topViewController];
+    [topViewController presentViewController:self.contentViewController animated:YES completion:nil];
```

Comment:
So, why not to merge if it's OK?
Also, Im thinking of creating Demo for iPhone with form designed for it. I.e. the one we had listed in issues.